scripts/run_all_benchmarks.sh

```diff
@@ -18,7 +17,7 @@ function run_benchmark() {
   $BENCHMARK \
   --pallet $1 \
   --extrinsic "*" \
-  --chain="$CHAIN" \
+  --chain="frequency-rococo" \
```

Comment:
This feels wrong.

Reply:
@wilwade  I have changed it to frequency , you may need to refesh